test/rebar_ct_SUITE.erl

```diff
@@ -339,6 +341,16 @@ multi_app_ct_macro(Config) ->
         true = lists:member({d, 'COMMON_TEST'}, ErlOpts)
     end, Apps).
 
+no_ct_suite(Config0) ->
```

Comment:
@ferd This test fails currently with `{error, interactive_mode}` while running the same setup from the command-line succeeds. I haven't found the culprit in the ct provider yet.

Reply:
Is this related to https://erlang.org/doc/apps/common_test/run_test_chapter.html#running-the-interactive-shell-mode ? I never managed to make that work properly or get anything to work with it since the instructions were kind of obscure.

It's possible that was failing way before this test case I figure.

Reply:
That is the one, although I don't see where the mode gets activated.

Reply:
Yeah I don't know. Curious if passing in a specific lack of options triggers the mode implicitly but I don't know why it would not trigger in the command-line vs others.

Reply:
Turns out running CT in an existing CT context is internally considered running in interactive mode. This explains why it works standalone. I adapted the test to only ensure that the options are properly set up.